pilot/pkg/model/push_context.go

```diff
@@ -695,6 +695,18 @@ func (ps *PushContext) ServiceForHostname(proxy *Proxy, hostname host.Name) *Ser
 	return nil
 }
 
+// ServicesForHostname returns a list of services that fall under the hostname provided. This hostname
+// can be a wildcard.
+func (ps *PushContext) ServicesForHostname(proxy *Proxy, hostname host.Name) []*Service {
```

Comment:
Why not use `ServiceForHostname(proxy *Proxy, hostname host.Name)` in push context?